changelog/unreleased/bug-fixes/2768--infer-non-config-types.md

```diff
@@ -0,0 +1,3 @@
+The `replace` and `extend` pipeline operators wrongly inferred address, subnet,
+pattern, and map values as strings. They are now inferred correctly. To force a
+value to be inferred as a string, wrap it inside double quotes.
```

Comment:
I tested with my recent react-eu example, and indeed now the replace value is of type address. However, "wrapping it in double quote" doesn't change that, so it's not showing up as string in my tests. Pipeline snippet:

```
      - replace:
          fields:
            src_ip: "1.2.3.4"
```

Reply:
You will have to escape those quotes, the one's you added are eaten by the YAML parser.

Reply:
that indeed works.